brian2/codegen/runtime/cython_rt/templates/spatialstateupdate.pyx

```diff
@@ -28,11 +27,13 @@
     cdef int _num_children
     cdef double _subfac
     cdef int _children_rowlength
+    cdef double *_v_previous = <double *>malloc(N * sizeof(double))
```

Comment:
I just realized that we should probably take the same approach as for the other temporary variables (e.g. `_ab_star0`, etc.), i.e. not allocate/free `_v_previous` every time step, but instead declare it as a "proper" variable in the model. This would mean adding it to the arrays [here](https://github.com/phreeza/brian2/blob/88e5372fc10878a16d5ac04bf520e2ed534f33cd/brian2/spatialneuron/spatialneuron.py#L375), and then referring to it via `{{_v_previous}}` in the template.